### ISSUE TITLE
TypeError: Cannot read property 'type' of undefined at PostgresQueryRunner.loadTables

### ISSUE THREAD

**Issue by osher-sade** [typeorm/typeorm#5747]
**Issue type:**

[ ] question
[x] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[x] `postgres`
[ ] `cockroachdb`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[x] `latest`
[ ] `@next`
[ ] `0.x.x`

**Steps to reproduce or a small repository showing the problem:**

Hello, this issue won't occur if you start typeorm when there are no tables to synchrozine with.
The error is thrown when there is at least one column of type `geometry` in any of the tables you tried synchroning with.
The error is being thrown [here](https://github.com/typeorm/typeorm/blob/master/src/driver/postgres/PostgresQueryRunner.ts#L1507).
The problem is caused by the fact that the query that you execute right before this line, returns empty result.
The result is empty because the last condition in that query is `"table_name" = '${table.name}'`, we can see that the `table.name` is being set [here](https://github.com/typeorm/typeorm/blob/master/src/driver/postgres/PostgresQueryRunner.ts#L1439).
The bottom line is that the data inside the column "table_name" in the table "geometry_columns" does not include schema name in it.
In order to solve this problem i think we should pass **just the table name** without the schema name included in it to the mentioned query.

**Comment by osher-sade** [comment]
I planned on opening a pull request, but I've noticed that the code responsible for that bug was pushed recently, in this [commit](https://github.com/typeorm/typeorm/commit/ee575570c72e7c45b286111e81fe5ee5d89dc021).

This made me wonder if I'm using outdated versions.
I'm running with PostgreSQL 10.11 and PostGIS 2.4.3

What versions of Postgres and PostGIS you ran with? @ohjeyong @ojy6042

**Comment by RobbieGM** [comment]
@osher-sade How are you working around this for the time being? One person suggested rolling back to 0.2.22 but that didn't work for me.
